src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+    private final ColorSupplier color = new ColorSupplier();
```

Comment:
```suggestion
    private final ColorSupplier colorSupplier = new ColorSupplier();
```
ця змінна не являється кольором, вона генерує кольори